src/components/ItaliaTheme/Blocks/Calendar/Item.jsx

```diff
@@ -79,26 +79,30 @@ const Item = ({ day, path, query, inEditMode, data }) => {
         {calendarDayResults[day] ? (
           calendarDayResults[day].items[day]?.map((item, index) => {
             const eventHasImage = Object.keys(item.image_scales).length > 0;
+            const item_start = new Date(item.start);
```

Comment:
siamo sicuri che così tiene conto del locale?

Reply:
da un test fatto si comportava in modo corretto, lascio aperto per verificare meglio.

Reply:
Qui vengono sempre giu tipi di dato evento e mi aspetterei che siano sempre tutte uguali quelle date in questo content-type. Proviamo a lasciare così